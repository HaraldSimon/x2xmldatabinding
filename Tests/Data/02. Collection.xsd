<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
	<xs:element name="Root">
		<xs:complexType>
			<xs:sequence>
				<xs:element name="ComplexParts">
					<xs:complexType>
						<xs:sequence>
							<xs:element name="ComplexPart" minOccurs="0" maxOccurs="unbounded">
								<xs:complexType>
									<xs:sequence>
										<xs:element name="ComplexValue" type="xs:string"/>
									</xs:sequence>
								</xs:complexType>
							</xs:element>
						</xs:sequence>
					</xs:complexType>
				</xs:element>
				<xs:element name="SimpleParts">
					<xs:complexType>
						<xs:sequence>
							<xs:element name="SimpleValue" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
						</xs:sequence>
					</xs:complexType>
				</xs:element>
				<xs:element name="InlinePart" type="xs:boolean" minOccurs="0" maxOccurs="unbounded"/>
                                <xs:element name="MultiCollection">
                                	<xs:complexType>
                                        	<xs:sequence>
                                                	<xs:element name="MultiItemFirst" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
                                                	<xs:element name="MultiItemSecond" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
                                                </xs:sequence>
                                        </xs:complexType>
                                </xs:element>
			</xs:sequence>
		</xs:complexType>
	</xs:element>
</xs:schema>
