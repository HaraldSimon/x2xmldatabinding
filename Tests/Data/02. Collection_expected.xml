<?xml version="1.0" encoding="UTF-8"?>
<DataBindingResult xsi:noNamespaceSchemaLocation="F:\Archive\2007\XMLDataBinding\Tests\XSD\DataBindingResult.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
	<Schemas>
		<Schema>
			<Name>02. Collection</Name>
			<Items>
				<Item>
					<Name>Root</Name>
					<ItemType>Interface</ItemType>
					<Interface>
						<InterfaceType>Interface</InterfaceType>
					</Interface>
				</Item>

        <!-- TODO: Parts moet worden gecollapsed naar een ItemType Collection -->
				<Item>
					<Name>ComplexParts</Name>
					<ItemType>Interface</ItemType>
					<Interface>
						<InterfaceType>Interface</InterfaceType>
					</Interface>
				</Item>

				<Item>
					<ItemType>Collection</ItemType>
					<Name>ComplexPartList</Name>
          <Collection>
            <ItemName>ComplexPart</ItemName>
          </Collection>
				</Item>

				<Item>
					<Name>ComplexPart</Name>
					<ItemType>Interface</ItemType>
					<Interface>
						<InterfaceType>Interface</InterfaceType>
					</Interface>
				</Item>


        <Item>
          <Name>SimpleParts</Name>
          <ItemType>Interface</ItemType>
          <Interface>
            <InterfaceType>Interface</InterfaceType>
          </Interface>
        </Item>

				<Item>
					<ItemType>Collection</ItemType>
					<Name>SimpleValueList</Name>
          <Collection>
            <ItemName>SimpleValue</ItemName>
          </Collection>
				</Item>
      </Items>
		</Schema>
	</Schemas>
</DataBindingResult>
